src/docs/sdk/event-payloads/index.mdx

```diff
@@ -159,6 +159,10 @@ For example, in a web app, this might be the route name.
 This value can be the git SHA for the given project, or a product identifier
 with a semantic version (suggested format `my-project-name@1.0.0`).
 
+It is suggested to attempt to automatically detect the release environment, 
+and set it automatically for the user. 
+Further inforomation about platform specific attempts here.
```

Comment:
```suggestion
Further information about platform specific attempts here.
```

Reply:
`here` was meant to link to the new page?

Reply:
yes, i didn't finish all changes yet. Should have left it in draft PR state. Need to move the rest of the information from notion and set up the links

Reply:
converted back to draft. feedback still much appreciated! 🖖